modules/nw-create-custom-ingress-controller.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-create-custom-ingress-controller_{context}"]
+= Creating a custom Ingress Controller
+
+As a cluster administrator, you can create a new custom Ingress Controller.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a custom ingress controller file.
+
+.. Create a file that defines the custom ingress controller custom resource (CR):
+
++
+.Example `custom-ingress-controller.yaml` file
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+    name: custom<1>
+    namespace: openshift-ingress-operator
+spec:
+defaultCertificate: <2>
+    name: custom-ingress-custom-certs
+domain: "custom-domain.mycompany.com"<3>
+----
+<1> Specify the `<name>` with a name for the `IngressController` object.
```

Comment:
```suggestion
<1> Specify a custom `name` for the `IngressController` object.
```

Reply:
oh!

Reply:
I definitely copied this straight from another procedure which probably needs work.